### PR TITLE
fix(emails): update download button to get started on sub welcome email

### PR DESCRIPTION
**Pull request by chenba**
Because:
 - not every product has a download step

This commit:
 - update the subscription welcome email's button label to Get Started

## Issue that this pull request solves

Closes: #12205

### CODE REVIEW

**Comment by chenba** [comment]
> Changes look like good. I tested both passwordless and regular flows.
> 
> Only question I have is whether this looks OK or not?
> 
What's the main difference between those buttons in the screenshots?  I'm afraid I'm not good enough to see it.

File: packages/fxa-auth-server/lib/senders/emails/templates/downloadSubscription/index.mjml [hunk 1]
```diff
@@ -19,23 +19,18 @@
...
-      <span data-l10n-id="downloadSubscription-link-action" data-l10n-args="<%= JSON.stringify({productName}) %>">
-        Download <%- productName %>
-      </span>
-    </mj-button>
-  </mj-column>
-</mj-section>
+<%- include ('/partials/button/index.mjml', {
+  buttonL10nId: "downloadSubscription-link-action-2",
+  buttonText: "Get started",
+  cssClass: 'mb-8'
```
[the remark this answers is not in the document]
Comment (chenba) [reply packages/fxa-auth-server/lib/senders/emails/templates/downloadSubscription/index.mjml:32]: That's something from Tailwind.

**Comment by StaberindeZA** [comment]
> > Changes look like good. I tested both passwordless and regular flows.
> > Only question I have is whether this looks OK or not?
> 
> What's the main difference between those buttons in the screenshots? I'm afraid I'm not good enough to see it.

The top image has a margin bottom of 32px, where the bottom image has no margin.

**Comment by chenba** [comment]
> 
> The top image has a margin bottom of 32px, where the bottom image has no margin.

Oh, I was too busy looking at the button itself.

File: packages/fxa-auth-server/lib/senders/emails/templates/downloadSubscription/index.mjml (hunk 1 above)
[the remark this answers is not in the document]
Comment (StaberindeZA) [reply packages/fxa-auth-server/lib/senders/emails/templates/downloadSubscription/index.mjml:32]: @chenba ah actually I take this back. I think leaving the `mb-8` class on this is fine.

Could we remove the `mt-5` class from this line though? I think that would get it in line with `subscriptionAccountFinishSetup`

`<%- include('/partials/subscriptionSupport/index.mjml', { cssClass: 'mt-5' }) %>`

[the remark this answers is not in the document]
Comment (chenba) [reply packages/fxa-auth-server/lib/senders/emails/templates/downloadSubscription/index.mjml:32]: Done

**Review by StaberindeZA — APPROVED** [review]
LGTM. Thank you for the quick change.

**Pull request merged**